app/controllers/home_controller.rb

```diff
@@ -1,4 +1,5 @@
 class HomeController < ApplicationController
```

Comment:
Yeap you can drop Home controller and replace by Ingidients controller with index show